cypress/integration/menubar.spec.js

```diff
@@ -0,0 +1,60 @@
+/// <reference types="cypress" />
+import {loadAndAcceptDisclaimer} from "../support/index"
+
+context('Top menu bar', () => {
```

Comment:
If Cypress offers mobile support, it'd be worth testing the mobile nav bar too - works on touch rather than mousehover.